mesheryctl/internal/cli/root/system/login.go

```diff
@@ -0,0 +1,49 @@
+package system
+
+import (
+	"io/ioutil"
+
+	"github.com/layer5io/meshery/mesheryctl/internal/cli/root/config"
+	"github.com/layer5io/meshery/mesheryctl/pkg/utils"
+	"github.com/pkg/errors"
+	"github.com/spf13/cobra"
+	"github.com/spf13/viper"
+
+	log "github.com/sirupsen/logrus"
+)
+
+var loginCmd = &cobra.Command{
+	Use:   "login",
+	Short: "Authenticate with meshery server",
+	Long: `
+Authenticate with meshery server
+
+The authetication mode is web-based browser flow`,
+	Args: cobra.MinimumNArgs(0),
+	RunE: func(cmd *cobra.Command, args []string) error {
+		mctlCfg, err := config.GetMesheryCtl(viper.GetViper())
+		if err != nil {
+			return errors.Wrap(err, "error processing config")
+		}
+
+		tokenData, err := utils.InitiateLogin(mctlCfg)
```

Comment:
Just before attempting this, we're going to want to check to see if `isMesheryRunning()`.

Reply:
Similar implementation to logout too.

Reply:
Fixed